cg/server/admin.py

```diff
@@ -640,6 +708,25 @@ class UserView(BaseView):
     edit_modal = True
 
 
+class IlluminaSampleSequencingMetricsView(BaseView):
+    column_list = [
+        "flow_cell",
+        "sample",
+        "flow_cell_lane",
+        "total_reads_in_lane",
+        "base_passing_q30_percent",
+        "base_mean_quality_score",
+        "_yield",
```

Comment:
```suggestion
        "yield",
```
Does this pick names from the DB table directly? In that case the name is just `yield`

Reply:
I am not sure, but if it doesn't I have to change this to `yield_` anyway. I will test this when the source branch is merged

Reply:
fixed to `yield_`